ui/app/src/@types/mui.d.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Chaos Mesh Authors.
```

Comment:
s/2021/2022/g

Reply:
Thx! I've already corrected both of them. 😃